avm/res/compute/virtual-machine/tests/e2e/windows.nvidia/main.test.bicep

```diff
@@ -35,7 +35,7 @@ var tempLocation = 'eastus'
 // =================
 resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
   name: resourceGroupName
-  location: tempLocation
+  location: resourceLocation
```

Comment:
Why was this changed from the hardcoded location? @rahalan I suggest to undo this in your next PR